cypress/support/config/services.js

```diff
@@ -786,9 +786,10 @@ module.exports = {
     pageTypes: {
       articles: {
         path:
+          // eslint-disable-next-line no-nested-ternary
```

Comment:
what was the reason to disable linting here ??
I don't think you need it

Reply:
Oh, shoot, had an earlier implementation that was nested, missed to remove the comment after i reverted. Thanks for taking note of this